frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -17,8 +17,8 @@ const HeaderSection = styled(Card)`
     top: 60px;
     position: sticky;
     z-index: 1;
-    background-color: ${tokens.colors.ui.background__light.hex};
     box-shadow: none;
+    background-color: ${tokens.colors.ui.background__light.hex};
 `
 const TitleSection = styled.div`
```

Comment:
Suggest putting the "Add to queue"-button under the title on smal screen (maybe flex: wrap?)
<img width="343" alt="image" src="https://github.com/user-attachments/assets/868b19fd-6182-407a-bd41-b2652ec78918">